Rehlate/index.html

```diff
@@ -0,0 +1,373 @@
+              ></textarea>
+            </span>
+              <button>أرسل الرسالة</button>
+            </form>
+          </div>
+          <ul>
+            <li>
+              <img
+                src="./assets/SVG/contact/help-icon.svg"
+                alt="contact-icon-01"
+              />
+              <div class="form-inline">
+              <h5 class="black">نقدم لك المساعدة</h5>
+              <p class="card-grey">
+                إن كان لديك اي مشكلة في البلد الذي تسافر إليه سنقوم بمساعدك في
+                هذا الموضوع.
+              </p>
+              </div>
+            </li>
+            <li>
+              <img
+                src="./assets/SVG/contact/communication-icon.svg"
+                alt="contact-icon-02"
+              />
+              <div class="form-inline">
+              <h5 class="black">للتواصل</h5>
+              <p class="card-grey">
+                يمكنك التواصل معنا عبر الفورم أو عبر البريد الإلكتروني:
+                <span class="purple">company@mail.com</span>
+              </p>
+              </div>
+            </li>
+            <li>
+              <img
+                src="./assets/SVG/contact/working-hours-icon.svg"
+                alt="contact-icon-03"
+              />
+              <div class="form-inline">
+              <h5 class="black">أوقات الدوام</h5>
+              <p class="card-grey">
+                يمكنك التواصل معنا يومياً أثناء أوقات الدوام من الساعة 9 صباحاً
+                حتى 6 مساءاً.
+              </p>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+    <footer class="black-bg ">
+      <div class="footer-inline">
+      
+        <div class="col1">
+            <div class="logo white">
+                <a href="index.html"><span class="yellow">.</span>
+                    رحلاتي</a>
+              </div>
+          <p class="body-grey">
+            شركة متخصصة في مجال السياحة والسفر نقدم خدمات تأمين الرحلات
+            والحجوزات والفيز لكل أنحاء العالم.
+          </p>
+        </div>
+        <div class="row">
+        <div class="col">
+          <h5 class="white">حول</h5>
+          <a>الشركة</a>
+          <a>من نحن؟</a>
+          <a>فرص العمل</a>
+          <a>المدونة</a>
+        </div>
+        <div class="col">
+          <h5 class="white">خدماتنا</h5>
+          <a>بطاقات الطيران</a>
+          <a>الرحلات البحرية</a>
+          <a>الفنادق</a>
+          <a>الرحلات البرية</a>
+          <a>تأمين الفيزا</a>
+        </div>
+        <div class="col">
+          <h5 class="white">الحجوزات</h5>
+          <a>العروض الجديدة</a>
+          <a>تأكد من حجزك </a>
+          <a>الاشتراكات</a>
+          <a>طرق السفر</a>
+        </div>
+        <div class="col">
+          <h5 class="white">الموقع</h5>
+          <a>سياسة الخصوصية</a>
+          <a>شروط الاستخدام</a>
+          <a>بلغ عن مشكلة</a>
+          <a>تواصل معنا</a>
+        </div>
+      </div>
+      </div>
+      <hr>
```

Comment:
Check the color of this `<hr>`; you can use `#191919` as it appears in the Figma design. You can search for how to style `<hr>` using the **border** property